pkg/storage/mto_solution.go

```diff
@@ -183,3 +183,24 @@ func MTOSolutionDelete(tx *sqlx.Tx, actorUserID uuid.UUID, _ *zap.Logger, milest
 	}
 	return nil
 }
+
+func MTOSolutionLinkMilestonesToSolutions(
```

Comment:
Similar to the message in resolvers, this is duplicating functionality (`MTOMilestoneSolutionLinkMilestonesToSolution`).